src/AppBundle/Resources/views/Report/Report/declaration.html.twig

```diff
@@ -48,7 +48,15 @@
         <div class="text">
             <p>{{ 'page.declarationTerms01' | trans }}</p>
             <p>{{ 'page.declarationTerms02' | trans }}</p>
-            <p>{{ 'page.declarationTerms03' | trans }}</p>
+            {% if report.isLayReport %}
```

Comment:
I wonder if this should be driven by the `user.role_name` instead of the report type. If we move towards having different user types accessing the same report (and potentially both being able to reach this declaration page), then we probably want to get away from using a hard coded report type to drive different behaviour based on who is logged in.